CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - [#320](https://github.com/clojure-emacs/clj-refactor.el/issues/320) `*data-readers*` ignored when searching for macros.
 - [#339](https://github.com/clojure-emacs/clj-refactor.el/issues/339) Teach stop refer to understand multiline refer clauses.
 - [#341](https://github.com/clojure-emacs/clj-refactor.el/issues/341) Avoid creating circular dependencies with move form when source namespace refers to the target namespace with a require.
+- Cygwin path fix. Conversion between Unix-like path generated by Emacs and Windows-like path required by nREPL.
```

Comment:
please reference the `refactor-nrepl` issue you created (see examples of this below in the changelog). also perhaps mention the fact that a cider solution is reused.